recipes/leftbar/style/leftbar_row.less

```diff
@@ -0,0 +1,206 @@
+.dt-leftbar-row {
+
+  // TODO: reconcile Theme colors, which probably means use them but
+  //       update https://dialpad.design/design/colors/#theme in dialtone.css
+  //       to reflect the new design
+
+  //  ============================================================================
+  //  $   CSS CUSTOM PROPERTIES
+  //  ----------------------------------------------------------------------------
+  --leftbar-row-color-foreground: var(--fc-primary);
+  --leftbar-row-color-background: transparent;
+  --leftbar-row-radius: 100em;
+  --leftbar-row-alpha-color-foreground: var(--fc-secondary);
+  --leftbar-row-alpha-width: calc(var(--size-300) * 10);
+  --leftbar-row-alpha-height: calc(var(--size-300) * 9);
+  --leftbar-row-omega-min-width: var(--leftbar-row-alpha-width);
```

Comment:
are greek letters like this just a way of referring to elements that "have no semantic name"?

Reply:
Yeah, abstracted. `alpha` and `omega` for "first" and "last" is a common CSS naming convention, especially BEM.

```html
<el class="dt-leftbar-row__primary"...>
  <div class="dt-leftbar-row__alpha"> ... </div>
  <div class="dt-leftbar-row__label"> ... </div>
  <div class="dt-leftbar-row__omega"> ... </div>
</el>
```